# angles are angle from lidar scan
# ranges are distance from lidar
# lengths are lengths of gap

float32[] r1
float32[] theta1
float32[] r2
float32[] theta2

float32[] x1
float32[] y1
float32[] x2
float32[] y2

float32[] euc_length
float32[] delta_angle